Modules/_ctypes/callproc.c

```diff
@@ -514,8 +522,8 @@ PyCArg_repr(PyCArgObject *self)
         break;
 
     default:
-        sprintf(buffer, "<cparam '%c' at %p>",
-            self->tag, self);
+        sprintf(buffer, "<cparam %02x' at %p>",
```

Comment:
Looks as if this change accidentally removed the opening apostrophe.
```suggestion
        sprintf(buffer, "<cparam '%02x' at %p>",
```

Reply:
Actually the opposite. I have forgot to remove the closing apostrophe.